### DISCUSSION TITLE
how to convert pnach file to opl cheat file

### DISCUSSION THREAD

**Discussion by PsxMicha** [PS2-Widescreen/OPL-Widescreen-Cheats#40] [General]
Hey. Could someone please share how to convert pnach to opl cheat file? 

Because I found  pnach for smash court tennis pro tournament 2 but I can't find it here in the list and I wanna convert it myself

**Comment by israpps** [comment]
If you have the mastercode please share it

So I can add the hack to this repository.

https://github.com/israpps/PS2-pnach-converter
